### ISSUE TITLE
Change Sphinx theme and update branding to match OFDS website

### ISSUE THREAD

**Issue by lgs85** [Open-Telecoms-Data/open-fibre-data-standard#139]
If time permits, we should try to create/adopt a sphinx theme that it isn't just the standard readthedocs branding.

**Comment by duncandewhurst** [comment]
Some themes to consider:

* [Nefertiti](https://sphinx-nefertiti.readthedocs.io/latest/index.html)
* [PyData](https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html)
* [Furo](https://pradyunsg.me/furo/)

I like Nefertiti. It's quite new, but seems comprehensive.

**Comment by duncandewhurst** [comment]
@stevesong do you have any preferences amongst the themes in my previous comment?

**Comment by stevesong** [comment]
No particular preference. They all seem reasonable. Happy to defer to your preference.

**Comment by duncandewhurst** [comment]
Great. I'll take a look at which is the best fit and can be most easily adapted to match the OFDS website branding.

**Comment by duncandewhurst** [comment]
@stevesong here is a lightly adapted version of the Nefertiti theme applied to the OFDS documentation: https://open-fibre-data-standard.readthedocs.io/en/139-nefertiti-theme/index.html.

So far, I only updated some fonts, the logo, and the header items. Let me know if you're happy with the Nefertiti theme in general, and I will make the necessary CSS updates to use the colour scheme from the OFDS website.

Alternatively, I had previously adapted the Furo theme, which you can preview here: https://open-fibre-data-standard.readthedocs.io/en/139-custom-theme/. Presently, that one is using ODS's colour scheme, but could be updated, if you prefer it to Nefertiti.

**Comment by duncandewhurst** [comment]
@stevesong are you happy to progress with the Nefertiti theme ([OFDS docs preview](https://open-fibre-data-standard.readthedocs.io/en/139-nefertiti-theme/index.html))? 

There's some more work to be done (colour palette and logo size), but I'll only do that if you're happy with the look/layout/navigation of the theme in general.

**Comment by stevesong** [comment]
I think this is fine.  Would be nice to invert the banner colours so that it looks more like OFDS.info and perhaps match default fonts.  There might be a small issue for the logo in "dark mode".  FWIW, I am happy to just remove the light/dark option.

**Comment by duncandewhurst** [comment]
I've updated the colours and disabled the light/dark mode switcher. I thought I'd already matched the fonts, but I'll double-check. Here's the latest [preview](https://open-fibre-data-standard.readthedocs.io/en/139-nefertiti-theme/index.html).

To do:

- [ ] ~~Check docs always default to light mode~~ (it was easier to fix the dark mode issues than to do this)
- [x] Check fonts and update if needed to match ofds.info
- [x] Get collapsible JSON examples working
- [x] Check all pages and screen sizes for CSS issues
- [x] Update copyright notice
- [x] Fix misaligned search box and button

<img width="1919" height="1332" alt="Image" src="https://github.com/user-attachments/assets/5ab1b9b2-9de5-463d-83c2-590bedd5ae0a" />

**Comment by stevesong** [comment]
This looks great!  I think my only question at this point is whether it makes sense to have the top level menu.  I think it creates some confusion between the standard documentation site and the main website.  I would keep the menu on the left hand side but keep the top menu empty except for a pointer back to https://ofds.info. Does that make sense?  Happy to discuss.

**Comment by duncandewhurst** [comment]
Good point. Fewer menu items also means that we can include the project name in the header - it's included by default, but I'd hidden it due to lack of space.

I've updated the theme so that the logo and project name link back to ofds.info (previously they linked to the index page of the documentation), added a header item to link to the documentation home page, and kept the CoVE link since it doesn't exist in the left-hand navigation menu: [preview](https://standard.ofds.info/en/139-nefertiti-theme/index.html#).

<img width="1920" height="1293" alt="Image" src="https://github.com/user-attachments/assets/96ace167-80de-45d1-9511-f2441ae691f0" />

Alternatively, we could change the project name to "Open Fibre Data Standard Documentation", have the logo and project name link to the index page of the documentation, and have a separate menu item to link to ofds.info.

<img width="1920" height="1293" alt="Image" src="https://github.com/user-attachments/assets/06383712-4d4e-4041-95fe-4de7ba36bf10" />

Let me know what you think.

**Comment by stevesong** [comment]
I think I prefer the former but I could be convinced either way.

**Comment by duncandewhurst** [comment]
@stevesong I think this now ready to deploy, which we can do without needing to release a new version of the standard itself. Please could you check that you're happy with the preview at: https://standard.ofds.info/en/139-nefertiti-theme/

Changes:

* Make logo and title link to docs homepage and add ofds.info to the navbar (I kept clicking the logo and title to get back to the docs homepage only to find myself on ofds.info, so I think this is the better option 
* Re-enable light/dark switcher (it was easier to fix the dark mode issues than it was to fully disable the switcher). Let me know if you spot any remaining issues in dark mode
* Fix collapsible JSON examples
* Fix misaligned search box and button

**Comment by duncandewhurst** [comment]
Theme applied to live docs site: https://standard.ofds.info/en/latest/.

I'll close this issue, but feel free to reopen if you find any issues with the new theme.
